src/main/java/calculator/app/Calculator.java

```diff
@@ -0,0 +1,23 @@
+package calculator.app;
+
+public class Calculator {
+
+    public Integer addition(Integer a, Integer b) {
+        return a + b;
+    }
+
+    public Integer subtraction(Integer a, Integer b) {
+        return a - b;
+    }
+
+    public Integer multiplication(Integer a, Integer b) {
+        return a * b;
+    }
+
+    public Double division(Integer a, Integer b) {
+        if (b == 0) {
+            throw new ArithmeticException("Division by zero");
+        }
```

Comment:
don't forget to divide by empty line logic blocks.

Check for Divide by Zero and Exception it's one thing
Return and divide other.